src/40select.js

```diff
@@ -368,9 +368,13 @@ yy.Select.prototype.compile = function(databaseid, params) {
 	// Now, compile all togeather into one function with query object in scope
 	var statement = function(params, cb, oldscope) {
 		query.params = params;
-		var res1 = queryfn(query, oldscope, function(res) {
-			//console.log(res[0].schoolid);
-			//console.log(184,res);
+		// Note the callback function has the data and error reversed due to existing code in promiseExec which has the

```

Comment:
Great with a good comment. I made the promiseExec myself - so I'm glad you found this bug!